changelog.d/README.md

```diff
@@ -0,0 +1,79 @@
+## Overview
```

Comment:
I'd suggest linking to this file from `CONTRIBUTING.md` since it is relevant context for contributors to be aware of.